base/common/logger_useful.h

```diff
@@ -42,6 +42,7 @@ namespace
 } while (false)
 
 
+#define LOG_TEST(logger, ...)    LOG_IMPL(logger, DB::LogsLevel::test, Poco::Message::PRIO_TEST, __VA_ARGS__)
```

Comment:
Interesting idea, maybe it should be under `#ifdef` so that only binaries on CI will enable those messages?

Reply:
We use our production binary in all `release` tests (and it looks reasonable). I'll think about defence here, maybe at least some runtime check (env variable)...

Reply:
I'm suggesting `#ifdef` here to make then literally no-op in relwithdebinfo/release builds (so that one will not care about additional cost for log messages, i.e. calling of some heavy method), but this has one downside - "variable maybe unused" warnings...

Reply:
@azat It costs only one `if`.
Macro arguments are lazy evaluated.